compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -494,6 +494,18 @@ class OMR_EXTENSIBLE CodeGenerator
    bool supportsMarshallingUnmarshallingIntrinsics() {return true;} // no virt, default
    bool supportsMergingOfHCRGuards() {return false;} // no virt, default
 
+   /** \brief
+    *     Determines whether this code generator supports concurrent scavange of objects during garbage collection.
+    *
+    *  \return
+    *     true if the code generator will emit read barriers for loads of object references from the heap in support
+    *     of concurrent scavange; false otherwise.
+    */
+   bool supportsConcurrentScavange()
```

Comment:
Typo, I think: isn't it scavenge not scavange?

Also, I wonder about the choice of "supports" as the verb here. This name suggests a question like "can this code generator operate with concurrent scavenge?" rather than "should I generate code for concurrent scavenge?". Maybe concurrentScavengeEnabled() or something like that?